packages/compass-components/src/components/tooltip.tsx

```diff
@@ -25,10 +26,12 @@ type TooltipTrigger = React.FunctionComponent;
 
 const Tooltip: React.FunctionComponent<
   TooltipTriggerProps &
-    Omit<LeafyGreenTooltipProps, 'open' | 'setOpen' | 'trigger'> & {
+    Omit<LeafyGreenTooltipProps, 'trigger'> & {
```

Comment:
Let us override 'open' | 'setOpen', used to change the interaction that triggers the tooltip: for example: show the "Copied!" tooltip after a Copy button is pressed.